src/prefect/core/flow.py

```diff
@@ -1225,6 +1226,18 @@ def serialize(self, build: bool = False) -> dict:
 
         return serialized
 
+    # Diagnostics  ----------------------------------------------------------------
+
+    def diagnostics(self, secrets: bool = True) -> str:
+        """
+        Get flow and Prefect diagnostic information
+
+        Args:
+            - secrets (bool, optional): toggle output of Secret names, defaults to True.
```

Comment:
This variable name isn't clear without really parsing the docstring -- I suggest making it more explicit (`include_secret_names`) or something.